src/components/Alert/AlertBanner.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+    <div :class="['banner-alert', {
+        'banner-alert--banner-danger': type === 'danger',
+        'banner-alert--banner-warning': type === 'warning',
+      }]">
+        <div v-show="text" class="banner-alert__container-text">
+          <unnnic-icon class="banner-alert__textIcon" v-show="isShowTextIcon(type)" :icon="getIconType(type)" size="sm" scheme="neutral-white" />
+          <span class="text" >{{ text }}</span>
+          <a v-if="linkHref" class="banner-alert__link" :href="linkHref" :target="linkTarget">
+            {{ linkText }}
+          </a>
+        </div>
+  
+        <div
+          v-show="showCloseButton"
+          class="banner-alert__close"
+          @click="emitClose"
+        >
+          <unnnic-icon icon="close" size="sm" scheme="neutral-white" />
+        </div>
+      </div>
```

Comment:
For better code readability and accessibility, use semantic elements instead of `div` and `span`